target/linux/ath79/dts/qca9533_tplink_tl-cpe210-v3.dts

```diff
@@ -7,8 +7,8 @@
 #include "qca9533.dtsi"
 
 / {
-	compatible = "tplink,tl-cpe-210-v2", "qca,qca9533";
-	model = "TP-LINK CPE210 v2";
+	compatible = "tplink,tl-cpe-210-v3", "qca,qca9533";
+	model = "TP-LINK CPE210 v3";
```

Comment:
There recently was a patch to rename from TP-LINK to TP-Link:
https://github.com/openwrt/openwrt/commit/79cc48ff608e0c9c9dcdf6197631783b9646ff82
I think you should stick to that.